.meteor/versions

```diff
@@ -84,7 +84,7 @@ livedata@1.0.18
 lmieulet:meteor-coverage@1.1.4
 localstorage@1.2.0
 logging@1.1.19
-lookback:emails@0.7.8
+lookback:emails@0.7.7
```

Comment:
Torolnod kell a packages directory-dban levo loocback: emails konyvtarat, hogy ismet az atmosphere-rol jovo verziot hasznald.
Kulonben a te verziot be lesz fixalva arra ami ott van nalad.